Snakefile

```diff
@@ -84,7 +84,7 @@ rule build_powerplants:
     input:
         base_network="networks/base.nc",
```

Comment:
why doesn't `base.nc` (and other associated, like `regions_onshore` etc.) go in RDIR as well? Depending on config, could be a different selection of countries etc. and could produce a different output...

Reply:
Compromise / practical consideration. I wanted to avoid the build_renewable_profiles is re-run for every new config file.